compiler/one-cmds/one-init

```diff
@@ -264,6 +337,23 @@ def _generate():
         config.write(f)
 
 
+def _get_model_type(parser, args):
+    if _utils._is_valid_attr(args, 'model_type'):
+        return args.model_type
+
+    if _utils._is_valid_attr(args, 'input_path'):
+        _, ext = os.path.splitext(args.input_path)
+
+        # ext would be, e.g., '.tflite' or '.onnx'.
+        # Note: when args.input_path does not have an extension, e.g., '/home/foo'
+        # ext after os.path.splitext() is '' and ''[1:] is still ''.
+        ext = ext[1:]
+        if ext in ["tflite", "onnx"]:
+            return ext
+
+    parser.error(f'the following argument is required: --input_path')
```

Comment:
`f'...'` is used but seems no variable used for format.

Reply:
if `ext` is not one of `tflite` nor `onnx` this error will be shown even if `--input_path` is given, which seems logical error.